app/controllers/api/v1/merchants/find_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::Merchants::FindController < ApplicationController
+
+  def show
+    render json: Merchant.find_by(find_params)
+  end
+
+  def index
+    render json: Merchant.where(find_params)
+  end
+private
+
```

Comment:
I think hound was yelling at us on the last project for not indenting "private".